### DISCUSSION TITLE
Project "group" property

### DISCUSSION THREAD

**Discussion by DejfCold** [alefragnani/vscode-project-manager#451] [Q&A] [answered]
Hi! This is possibly related to #50 

Is the `group` property used in any way? I expected it to work like the `FAVORITES`, `GIT` and `HELP AND FEEDBACK` tabs on the sidebar, but it didn't work that way and I went to look how does it work then. Didn't find anything other than the issue mentioned above. I noticed you've used a `groups` array instead of the `group` string that gets created for every project.

I assume this doesn't work yet.

```json
...
	{
		"name": "Email Client Electron",
		"rootPath": "C:\\xxx\\EmailClient\\electron",
		"paths": [],
		"group": "Email Client",
		"enabled": true
	},
	{
		"name": "Email Client Front-end",
		"rootPath": "C:\\xxx\\EmailClient\\front-end",
		"paths": [],
		"group": "Email Client",
		"enabled": true
	},
	{
		"name": "Email Client backend",
		"rootPath": "C:\\xxx\\EmailClient\\backend",
		"paths": [],
		"group": "Email Client",
		"enabled": true
	},
...
```

**Comment by alefragnani** [comment, marked as answer]
Hi @DejfCold ,

Yes, that `group` property is still not being used, and will be related to #50. In fact, I'm thinking about using another name, maybe _tag_ or _label_, I'm not sure, mostly because there is a _groupList_ setting which has no meaning to this. In the end, the idea is simple, be able to organize (treeview like) your projects by this group/tag/label.

On the other hand, the `projectManager.groupList` setting group/organize the projects presented in the `Command Palette`, just to sort them in a way to present each _kind of project_ separately. 

I'm planning to work on #50 soon, and will probably be a multi-release work. I'll update the issue with more details, once I start to work  on it.

Hope this helps
